src/src/beneficiaries/beneficiaries.controller.ts

```diff
@@ -100,6 +136,29 @@ export class BeneficiariesController {
 		return this.beneficiariesService.getStatuswiseCount(request, response);
 	}
 
+	@Get('ip-duplication')
```

Comment:
instead of 2 apis can we create one like this?
admin/list/duplicates-count-by-aadhaar?role=staff / program_owner

Reply:
Created single API